en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -207,16 +207,101 @@ For more information on initializing an API Project using OpenAPI/Swagger Specif
+        ```bash
+        $ apictl vcs status -e prod
+        Projects to Deploy (1)
+
+        APIs (1) ...
+        1: [save]		SwaggerPetstore-1.0.0: (SwaggerPetstore-1.0.0)
+        ```
+
+3.  Import the **SwaggerPetstore** API into the production environment by running the following sample command.
 
     !!! tip
         - Make sure you have already logged-in to the `prod` environment. For more information, see 
         [Login to an Environment]({{base_path}}/learn/api-controller/getting-started-with-wso2-api-controller/#login-to-an-environment).
 
         - A user with `admin` role is allowed to import APIs. To create a custom user who can import APIs, refer [Steps to Create a Custom User who can Perform API Controller Operations]({{base_path}}/learn/api-controller/advanced-topics/creating-custom-users-to-perform-api-controller-operations/#steps-to-create-a-custom-user-who-can-perform-api-controller-operations).
 
+    !!! example
+        ```bash
+        $ apictl vcs deploy -e prod
+        Deploying Projects (1)...
+
+        APIs (1) ...
+        1: SwaggerPetstore-1.0.0: (SwaggerPetstore-1.0.0)
+        Successfully imported API
+        ```
+
+    Now the tool will automatically detect the target environment and prepare a new artifact containing environment-related details. 
+
+    If you run `vcs deploy` command again, you will see the below output indicating that the deployment is already up-to-date.
+
+    !!! example
+        ```bash
+        $ apictl vcs deploy -e prod
+        Everything is up-to-date
+        ```
+
+    !!! tip
+        If you have multiple APIs to promote through CI/CD, you can commit those API projects as well to same repository. The rest of the process will be the same as above.
+ 
+        If you have export (or initiallized) and commited another API **Pizzashack-1.0.0** to the same repository and then run `vcs status` and `vcs deploy` commands, you will be able to see the below outputs. 
+
+        ```bash
+        $ apictl vcs status -e prod
+        Projects to Deploy (1)
+
+        APIs (1) ...
+        1: [save]		Pizzashack-1.0.0: (Pizzashack-1.0.0)
+
+        $ apictl vcs deploy -e prod
+        Deploying Projects (1)...
+
+        APIs (1) ...
+        1: Pizzashack-1.0.0: (Pizzashack-1.0.0)
+        Successfully imported API
+        ```
+
+        Here, **apictl** detects that there's a new API project is available to deploy. Without re-deploying the other unchanged API **SwaggerPetstore-1.0.0**, it will deploy only the new API.
+
+    !!! important
+        For deploying an API using `vcs deploy` command: 
+        
+        -   It is mandatory to have your API projects in a GIT based version control system.
+        -   It is mandatory to have `api_params.yaml` file inside each API Project. This is created by default when you exported an API using `export-api` or initiallized an API Project using `init`. In `api_params.yaml`. The below configuration section is used to deploy the API.
```

Comment:
Can we change Initiallized as initialized?

Reply:
Fixed